### DISCUSSION TITLE
Duplicated Attribute in Validation Event Documentation

### DISCUSSION THREAD

**Discussion by finis-fox** [israel-dryer/ttkbootstrap#487] [Q&A]
In the [API documentation](https://ttkbootstrap.readthedocs.io/en/latest/api/validation/#ttkbootstrap.validation.ValidationEvent) for the Validation Module, there's a duplicate "insertnamedelete" attribute for the inserted/deleted index and the inserted/deleted text in the ValidationEvent class.

When I tried ValidationEvent.\_\_dict__ , the "insertnamedelete" attribute gave the inserted/deleted text as its value, and there was no attribute giving the index of the inserted/deleted text.

Am I misunderstanding something or is the documentation wrong?

**Comment by TheDemonGuard** [comment]
@rurirurii 
It seems the section you mentioned in the documentation is fixed already.
[https://ttkbootstrap.readthedocs.io/en/latest/api/validation/#ttkbootstrap.validation.ValidationEvent](https://ttkbootstrap.readthedocs.io/en/latest/api/validation/#ttkbootstrap.validation.ValidationEvent)

> In the [API documentation](https://ttkbootstrap.readthedocs.io/en/latest/api/validation/#ttkbootstrap.validation.ValidationEvent) for the Validation Module, there's a duplicate "insertnamedelete" attribute for the inserted/deleted index and the inserted/deleted text in the ValidationEvent class.
> 
> When I tried ValidationEvent.__dict__ , the "insertnamedelete" attribute gave the inserted/deleted text as its value, and there was no attribute giving the index of the inserted/deleted text.
> 
> Am I misunderstanding something or is the documentation wrong?
